lab6.md

```diff
@@ -123,6 +123,36 @@ $ faas-cli up -f show-html.yml
 
 Open your browser and access http://127.0.0.1:8080/function/show-html. You should see a "Here's a new page!" HTML page rendered in the browser.
 
+Now try how it works with a path.
```

Comment:
`Now try how it works with a path.`

This sentence isn't correct, can you re-write it please?

Reply:
@ivanayov please could you rephrase, thank you :slightly_smiling_face: